### PR TITLE
YLP-1871 Update client to retrieve sent invitations and to invite a hcp in a team

### PR DESCRIPTION
**Pull request by TimCanu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Hydrophone is the module responsible for sending emails.
 This API sends notifications to users for things like forgotten passwords, initial signup, and invitations.
 
+## 1.12.2 - 2022-10-25
```
Comment (Nas84) [inline CHANGELOG.md]: Move than a fix, it should be 1.13 (yep we decide to be strict on the semver for go project)

File: clients/hydrophone/hydrophone_client.go [hunk 2]
```diff
@@ -91,6 +97,62 @@ func (client *Client) GetPendingInvitations(userID string, authToken string) ([]
...
+		return &retVal, nil
+	}
+	return nil, handleErrors(res, req)
+}
+
+func (client *Client) GetSentInvitations(ctx context.Context, userID string, authToken string) ([]models.Confirmation, error) {
+	logger := appContext.GetLogger(ctx)
+	req, err := client.getFullRequestWithContext(ctx, "GET", authToken, nil, map[string]string{}, "invite", userID)
+	if err != nil {
+		return nil, errors.Wrap(err, "SendTeamInviteHCP: error formatting request")
```
Comment (Nas84) [inline clients/hydrophone/hydrophone_client.go]: is it really "SendTeamInviteHCP" copy paste error ?

**Review by Nas84 — CHANGES_REQUESTED** [review]

File: clients/hydrophone/hydrophone_test.go [hunk 3]
```diff
@@ -80,6 +83,74 @@ func TestGetPendingInvitations(t *testing.T) {
 	testError(t, hydrophoneClient, testToken, "invite")
 }
 
+func TestGetSentInvitations(t *testing.T) {
```
Comment (Nas84) [inline clients/hydrophone/hydrophone_test.go:91]: running with coverage show that it misses the errors cases

**Review by Nas84 — CHANGES_REQUESTED** [review]

File: clients/hydrophone/hydrophone_test.go (hunk 3 above)
Comment (TimCanu) [reply clients/hydrophone/hydrophone_test.go:91]: There is an error case that I don't know how to reach... If you can show me that would be great :slightly_smiling_face:

**Review by Nas84 — APPROVED** [review]
LGTM

**Pull request merged**